pkg/preparer/listener.go

```diff
@@ -191,13 +197,14 @@ func (l *HookListener) writeHook(event string, hookPod *pods.Pod, manifest *pods
 		}
 
 		for _, executable := range executables {
+			executable.Cgexec = "" // do not cgexec in a hook
```

Comment:
I'm kind of confused by this comment but I realized there's a good point in there regarding where we should set `cgexec=""`, will fix in a second